source/geod24/LocalRest.d

```diff
@@ -259,8 +259,8 @@ private bool is_main_thread;
     usually provided by `vibe.core.core : runTask`.
 
     In order for them to properly work, we need to integrate them to our event
-    loop by providing the ability to spawn a task, and wait on some condition,
-    optionally with a timeout.
+    loop by providing the ability to spawn a task, and wait and timer
+    on some condition, optionally with a timeout.
```

Comment:
That sentence does not compute

Reply:
Deleted the existing phrase and then Function document added